dll/win32/shell32/lang/de-DE.rc

```diff
@@ -37,7 +37,7 @@ BEGIN
         MENUITEM "Aktualisieren", FCIDM_SHVIEW_REFRESH
```

Comment:
```suggestion
        MENUITEM "A&ktualisieren", FCIDM_SHVIEW_REFRESH
```
I don't really care whether you apply this specific suggestion, because every sane human being would simply use F5 for that, instead of using the menu-option with the accelerator. So not having this accelerator doesn't really hurt anybody. But I just did this suggestion to show you that your general philosophy of your changes to strip the accelerators for this menu are not correct. I attached 2 pictures from Windows for proof.